api/scripts/add-many-students-to-sco-certification-center.js

```diff
@@ -0,0 +1,56 @@
+const { MIDDLE_SCHOOL_ID } = require('../db/seeds/data/organizations-sco-builder');
+const times = require('lodash/times');
+const faker = require('faker');
+const SchoolingRegistration = require('../lib/domain/models/SchoolingRegistration');
+const { SchoolingRegistrationsCouldNotBeSavedError } = require('../lib/domain/errors');
+const { knex } = require('../lib/infrastructure/bookshelf');
+const DomainTransaction = require('../lib/infrastructure/DomainTransaction');
+
+function buildSchoolingRegistration() {
```

Comment:
Proposition : préfixer les fonctions définies dans ce fichier par `-` (sauf `main()`) pour qu'on les différencie des fonctions importées :
```suggestion
function _buildSchoolingRegistration() {
```